Formula/liquigraph.rb

```diff
@@ -1,11 +1,13 @@
 class Liquigraph < Formula
   desc "Migration runner for Neo4j"
   homepage "https://www.liquigraph.org/"
-  url "https://github.com/liquigraph/liquigraph/archive/liquigraph-4.0.4.tar.gz"
-  sha256 "2ccadb61da07ede6860c32695853456e4844eeb5e64c05f3e61b3c914587a072"
+  url "https://github.com/liquigraph/liquigraph/archive/liquigraph-4.0.6.tar.gz"
+  sha256 "c51283a75346f8d4c7bb44c6a39461eb3918ac5b150ec3ae157f9b12c4150566"
   license "Apache-2.0"
   head "https://github.com/liquigraph/liquigraph.git", branch: "4.x"
 
+  deprecate! date: "2022-02-21", because: "Check out `liquibase` and its Neo4j plugin instead"
```

Comment:
This should be after the `bottle do` block
```suggestion
  deprecate! date: "2022-02-21", because: :unsupported
```